sites/friday/src/manage-resources/resource-init.md

```diff
@@ -13,33 +13,102 @@ keywords:
+unless you set another [resource initialization strategy](#set-a-resource-initialization-strategy).
+
+You can also use a specific resource initialization strategy when performing certain actions,
+such as [branching](#environment-branch), [merging](#environment-merge),
+and [activating](#environment-activation) an environment, or [restoring a backup](#backup-restoration).
+
+## Default resources
+
+{{% vendor/name %}} allocates the following default resources to every container when deployed for the first time:
 
 | Resource type               | Amount |
 | --------------------------- | ----------- |
 | CPU                         | 0.5 |
 | RAM                         | Depends on the [container profile](/manage-resources/adjust-resources.md#advanced-container-profiles). |
 | Disk size (only applicable if the app or service requires a disk)                   | 512 MB |
 
-If you don't want to use those default resources, you can define a different [resource initialization strategy](/manage-resources/resource-init.md#define-a-resource-initialization-strategy).
-You can also [adjust those resources](/manage-resources/adjust-resources.md) after your project or new container has been deployed.
+If you don't want to use these default resources, you can set another [resource initialization strategy](/manage-resources/resource-init.md#set-a-resource-initialization-strategy).
+You can also [adjust resources](/manage-resources/adjust-resources.md) after your project or new container has been deployed.
+
+{{% note %}}
 
 For information on costs related to resource usage, see the [{{% vendor/name %}} pricing page](https://upsun.com/pricing/).
 Note that you can [keep an eye on those costs](/manage-resources/resource-billing.md) in the Console.
 
-## Define a resource initialization strategy
+{{% /note %}}
 
-You can define which strategy {{% vendor/name %}} uses to allocate resources
-when you first deploy your project or add a new container.
-The following strategies are available:
+
+## Set a resource initialization strategy
+
+{{% vendor/name %}} provides the following resource initialization strategies:
 
 | Strategy | Description |
 | ---------| ----------- |
-| `default`  | Initializes the new containers using the [{{% vendor/name %}} default resources](/manage-resources/resource-init.md).</br>This is the strategy that applies when you first deploy your project or new container, unless you explicitly set another strategy.  |
+| `default`  | Initializes the new containers using the [{{% vendor/name %}} default resources](#default-resources).</br>This strategy applies when you first deploy your project (or new container) unless you explicitly set another strategy, or allocate resources manually via the `resources:set` CLI command.  |
 | `manual`   | With this strategy, the first deployment fails and you need to configure resources manually through [the Console](/manage-resources/adjust-resources.md), or using `resources:set` in the CLI. |
```

Comment:
For the manual strategy, I think it would be nice to add also something explaining why this strategy could be a good option for the user, depending on their use case.
Because with the current description, with the failure etc., it doesn't sound very appealing 😅
The benefit of this strategy is that you can directly deploy with the desired resources. Whereas with the other strategies, if it's not exactly the resources that you want/need, you need to do a second deployment to change them. Which means that your env will run for a short moment with not exactly the wished resources. And the 2 deployments both involve some downtime (until we have real zero downtime deployments).